contracts/modules/cover/Cover.sol

```diff
@@ -53,8 +53,8 @@ contract Cover is MasterAware {
   );
```

Comment:
Index `uint coverId` as well

Reply:
Should we index `coverAsset` as well? So we can filter quickly like "fetch all DAI covers"...?

Reply:
the limitation here is only 3 indexed fields i already have 3 

    address indexed buyer,
    address indexed contractAddress,
        CoverType indexed coverType,
        
        
        can swawp coverType out for coverAsset would make sense, but can't do more than that